### DISCUSSION TITLE
关于播放器右边的弹幕列表

### DISCUSSION THREAD

**Discussion by Elysia-6** [the1812/Bilibili-Evolved#1977] [功能建议 / Ideas]
播放器右边的**弹幕列表**总是展开，可以做个自动收起或隐藏吗
有一些**多p视频**想切换选集的时候（尤其是**看课件**的时候）总是展开
体验不是很好，如果无法自动收起那可以把多p往上面调调试试？
当然不是必须要，只是自己闲麻烦，希望作者大大稍微考虑一下，感谢
![image](https://user-images.githubusercontent.com/58381825/122759735-ddb94500-d2cc-11eb-9d81-0a3cef807cd3.png)
ps：单p不影响，多p太需要了，尤其是一个系列

**Comment by Geeyun-JY3** [comment]
现在B站默认不就是收起的吗？把脚本相应功能关掉就是了。

**Reply by Elysia-6** [reply]
我看到了，是我瞎了，emmmm
